### DISCUSSION TITLE
Auth Helpers: Getting ERR_MODULE_NOT_FOUND due to importing without file extensions

### DISCUSSION THREAD

**Discussion by jamalmoir** [supabase/supabase#9185] [Questions] [answered]
I'm trying to use the sveltekit helpers and am getting errors like the following for all imports:
`Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/.../node_modules/@supabase/auth-helpers-sveltekit/dist/config' imported from /Users/.../node_modules/@supabase/auth-helpers-sveltekit/dist/server/utils/cookies.js at new NodeError (node:internal/errors:393:5) at finalizeResolution (node:internal/modules/esm/resolve:305:11) at moduleResolve (node:internal/modules/esm/resolve:866:10) at defaultResolve (node:internal/modules/esm/resolve:1074:11) at nextResolve (node:internal/modules/esm/loader:163:28) at ESMLoader.resolve (node:internal/modules/esm/loader:838:30) at ESMLoader.getModuleJob (node:internal/modules/esm/loader:425:18) at ModuleWrap. (node:internal/modules/esm/module_job:76:40) at link (node:internal/modules/esm/module_job:75:36)`

I believe this to be because of `"type": "module",` in my package.json and imports not using the .js extensions inside of this library.

How can I get around this?
`Node.js v18.9.1` (also tried 16.15.0)
`"@supabase/auth-helpers-sveltekit": "^0.7.0",`
`"@supabase/supabase-js": "^1.35.6",`
`"@sveltejs/kit": "^1.0.0-next.504",`

**Comment by silentworks** [comment, marked as answer]
This should be reported on the repo, I'm currently working on this issue as this was released last night. Until we officially announce its release you can expect breaks.
